pycm/generate_random_data.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+"""This file contains a function to generate a random confusion matrix."""
+import numpy as np
+from enum import Enum
+
+
+class Scenario(Enum):
```

Comment:
The name `Scenario` is too generic, which could lead to naming conflicts or confusion.

I suggest these alternatives:

1. `ClassDistributionScenario`
2. `ClassDistribution`
3. `ClassBalancePattern`

@sadrasabouri @alirezazolanvari

Reply:
2nd one is the best I think but again generic, if so 1st one if my choice.

Reply:
I guess we will use the second one in planned future features. So, I prefer the first one